packages/config-loader/src/loader.ts

```diff
@@ -126,8 +126,16 @@ export async function loadConfig(
     .filter((e): e is { url: string } => e.hasOwnProperty('url'))
     .map(configTarget => configTarget.url);
 
-  if (remote === undefined && configUrls.length > 0) {
-    throw new Error(`Remote config detected but this feature is turned off`);
+  if (remote === undefined) {
+    if (configUrls.length > 0) {
+      throw new Error(
+        `Remote config detected but this feature is turned off. Please enable by passing remote option in loadBackendConfig() call inside packages/backend/src/index.ts. See https://backstage.io/docs/conf/writing#configuration-files for detailed info.`,
+      );
+    }
+  } else if (remote.reloadIntervalSeconds === undefined) {
```

Comment:
We generally don't guard against things that are caught by TypeScript, except if there's a particular reason. Let's let TypeScript catch this one.

Some to think of it though, do we want to add support for disabling the reloading if `0` is passed? I could see that being a useful feature and one that currently will lead to some pretty bad behavior 😅